### ISSUE TITLE
Animation does not work

### ISSUE THREAD

**Issue by hasratyan** [michalsnik/aos#89]
Hello. I attached all components to my html file, but on scroll, there is not working any effect.

**Comment by hasratyan** [comment]
`		<main>
			<section id="wrapper">
				<video autoplay loop muted preload="auto">
					<source src="video.mp4" type="video/mp4">
				</video>
				<h1 class="headline slide">
					<span>Մենք ուսուցանում ենք</span>
					<span class="cd-words-wrapper">
						<b class="is-visible">Վեբ ծրագրավորում</b>
						<b>JAVA ծրագրավորում</b>
						<b>C# ծրագրավորում</b>
						<b>Python ծրագրավորում</b>
						<b>Android ծրագրավորում</b>
					</span>
				</h1>
				<a href="#">Ի՞նչ է ծրագրավորումը</a>
			</section>
			<section id="main" >
				<section id="benefites">
					<article>
						<h1>Ծրագրավորում սովորելու լավագույն վայրը</h1>
					</article>
					<article>
						<h1>Ավելի քան 130 շրջանավարտ</h1>
					</article>
					<article>
						<h1>Աշխատանքի առաջարկ ուսման ընթացքում</h1>
					</article>
					<article>
						<h1>Պատրաստի նախագծեր</h1>
					</article>
				</section>
				<div data-aos="zoom-in">Heeeey</div>
			</section>
		</main>
		<footer>
			<iframe src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2Fbasic.it.center%2F&tabs&width=340&height=214&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=true&appId" width="340" height="214" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
		</footer>
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
		<script src="https://cdn.rawgit.com/michalsnik/aos/2.1.1/dist/aos.js"></script>
		<script>
				 AOS.init({
        easing: 'ease-in-out-sine',
         duration: 1200
      });`

**Comment by mirceatrancota** [comment]
Try adding this:
`<script>
AOS.init({
	easing: 'ease-in-out-sine'
});
</script>`
In the footer after you loades aos.js

**Comment by hasratyan** [comment]
I have added it, but anyway, it does not work.

`<!DOCTYPE html>
<html lang="am-hy">
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<title>Basic IT Center | Ծրագրավորման դասընթացներ</title>
		<link href="https://cdn.rawgit.com/michalsnik/aos/2.1.1/dist/aos.css" rel="stylesheet">
		<link rel="stylesheet" href="styles/main.css">
	</head>
	<body>
		<a href="#cd-nav" class="cd-nav-trigger">Menu 
			<span class="cd-nav-icon"></span>
		</a>	
		<div id="cd-nav" class="cd-nav">
			<div class="cd-navigation-wrapper">
				<div class="half-block">
					<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12192.154114461138!2d44.523508!3d40.185956!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x3a1d7a8e8fbc0efe!2sBasic+IT+Center!5e0!3m2!1shy!2s!4v1483282369191" frameborder="0" style="border:0" allowfullscreen></iframe>
				</div>
				<div class="half-block">
					<nav>
						<ul class="cd-primary-nav">
							<li><a href="#0" class="selected" data-hover="Գլխավոր">Գլխավոր</a></li>
							<li><a href="#0" data-hover="Դասընթացներ">Դասընթացներ</a></li>
							<li><a href="#0" data-hover="Մեր մասին">Մեր մասին</a></li>
							<li><a href="#0" data-hover="Աշխատանք">Աշխատանք</a></li>
							<li><a href="#0" data-hover="Հետադարձ կապ">Հետադարձ կապ</a></li>
						</ul>
					</nav>
				</div>
			</div>
		</div>
		<header>
			<a href="/" id="logo"><img src="images/logo.svg"></a>
			<nav>
				<a href="#"><span data-hover="Դասընթացներ">Դասընթացներ</span></a>|
				<a href="#"><span data-hover="Հետադարձ կապ">Հետադարձ կապ</span></a>|
				<a href="#"><span data-hover="Մուտք">Մուտք</span><img src="images/icos/login-arrow.png" style="    height: 17px;
    vertical-align: middle;
    margin-left: 5px;
    padding-bottom: 1px;"></a>
			</nav>
		</header>
		<main>
			<section id="wrapper">
				<video autoplay loop muted preload="auto">
					<source src="video.mp4" type="video/mp4">
				</video>
				<h1 class="headline slide">
					<span>Մենք ուսուցանում ենք</span>
					<span class="cd-words-wrapper">
						<b class="is-visible">Վեբ ծրագրավորում</b>
						<b>JAVA ծրագրավորում</b>
						<b>C# ծրագրավորում</b>
						<b>Python ծրագրավորում</b>
						<b>Android ծրագրավորում</b>
					</span>
				</h1>
				<a href="#">Ի՞նչ է ծրագրավորումը</a>
			</section>
			<section id="main" >
				<section id="benefites">
					<article>
						<h1>Ծրագրավորում սովորելու լավագույն վայրը</h1>
					</article>
					<article>
						<h1>Ավելի քան 130 շրջանավարտ</h1>
					</article>
					<article>
						<h1>Աշխատանքի առաջարկ ուսման ընթացքում</h1>
					</article>
					<article>
						<h1>Պատրաստի նախագծեր</h1>
					</article>
				</section>
				<div data-aos="zoom-in">Heeeey</div>
			</section>
		</main>
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
		<script src="https://cdn.rawgit.com/michalsnik/aos/2.1.1/dist/aos.js"></script>
		<script>
				 AOS.init({
        easing: 'ease-in-out-sine',
         duration: 1200
      });
		</script>
	</body>
</html>`

**Comment by hasratyan** [comment]
I have added, but anyway, it does not work.

**Comment by michalsnik** [comment]
Hey, please take a look at http://codepen.io/michalsnik/pen/WxNdvq and make sure you're loading and executing AOS properly. It simply has to work, when used correctly :)

**Comment by hasratyan** [comment]
I everything did correctly using the instruction of setup of AOS. When I add attribute `data-aos="zoom-in"` , it automatically adds the class `item aos-init` , but on scrolling nothing happens.

**Comment by michalsnik** [comment]
Have you been playing with scroll somehow? The position is calculated properly - when you call `AOS.refresh()` in console it returns array of nodes that should animate with their trigger point. The problem is that `window.scrollY` always returns 0 on your page, thus AOS can't detect if page was scrolled.

**Comment by hasratyan** [comment]
I found the problem. In CSS I used height:100% for body and html. That was a problem

**Comment by michalsnik** [comment]
Oh, yes. I'm glad you figured it out. I guess we can close this issue then :)
